next.config.js

```diff
@@ -1,4 +1,4 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
-  reactStrictMode: true,
+  reactStrictMode: false,
```

Comment:
Latest react version produces 2 onMount events in strict mode. Disabled to save time fixing it